cdap-docs/reference-manual/source/http-restful-api/metrics.rst

```diff
@@ -65,7 +65,15 @@ Available Contexts
 The context of a metric is typically enclosed into a hierarchy of contexts. For example,
 the flowlet context is enclosed in the flow context, which in turn is enclosed in the
 application context. A metric can always be queried (and aggregated) relative to any
-enclosing context. These are the available application contexts of CDAP:
+enclosing context. 
+
+The *consumer* context relates to entities that are a recipient of events, typically a
+flowlet. A consumer flowlet is the recipient of a queue. Conversely, *producers* are
+entities that emits events, such as a stream or a flowlet. Flowlets can at the same time
+be both consumers and producers. The difference between the total of the producers and the
```

Comment:
same comment as above. its the difference between produced events and consumed events. not difference of the producers and consumers isn't it?